### PR TITLE
Feat reset password

### PR DESCRIPTION
**Pull request by EnzoHuynh**

### CODE REVIEW

File: src/Entity/User.php [hunk 1]
```diff
@@ -28,6 +31,7 @@ class User implements UserInterface
 
     /**
      * @ORM\Column(type="string", length=30, unique=true)
+     * @Assert\NotBlank(message="test")
```
Comment (dycor) [inline src/Entity/User.php]: ??????

File: src/Form/ResetPasswordType.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<?php
```
Comment (taingk) [inline src/Form/ResetPasswordType.php]: Delete

File: templates/front/profile/index.html.twig [hunk 3]
```diff
@@ -19,6 +19,10 @@
         </tbody>
     </table>
 
+    <div>
```
Comment (taingk) [inline templates/front/profile/index.html.twig]: Delete

File: src/Controller/front/ProfileController.php [hunk 4]
```diff
@@ -22,4 +31,37 @@ public function show(User $user): Response
     {
         return $this->render('front/profile/index.html.twig', ['user' => $user]);
     }
+
+    public function resetPassword(Request $request, User $user)
```
Comment (taingk) [inline src/Controller/front/ProfileController.php]: Delete

File: templates/front/profile/index.html.twig (hunk 3 above)
Comment (taingk) [reply templates/front/profile/index.html.twig]: Déplace le fichier templates/front/security/admin.html.twig dans templates/back/security/admin.html.twig pls

**Pull request merged**